src/pluto/operation/rotate.cr

```diff
@@ -0,0 +1,100 @@
+module Pluto::Operation::Rotate
+  def rotate(
+    degrees : Float64,
+    *,
+    center_x : Int32 = width // 2,
+    center_y : Int32 = height // 2,
+    radius : Int32 = -1,
+    pad : Bool = false,
+    pad_type : PaddingType = PaddingType::Black
+  ) : self
+    clone.rotate!(degrees, center_x: center_x, center_y: center_y, radius: radius, pad: pad, pad_type: pad_type)
+  end
+
+  # Can't really find good ways to reduce this more than it is, so disabling this check here
+  # ameba:disable Metrics/CyclomaticComplexity
+  def rotate!(
+    degrees : Float64,
+    *,
+    center_x : Int32 = width // 2,
+    center_y : Int32 = height // 2,
+    radius : Int32 = -1,
+    pad : Bool = false,
```

Comment:
Please replace all mentions of "pad" with "padding".

Reply:
Doh! Can do :+1: